### ISSUE TITLE
Dark Angel Deathwing Companions Issue

### ISSUE THREAD

**Issue by warpspyder** [BSData/horus-heresy#2317]
**Describe the bug**
When selecting a combi weapon of any type it makes the entire unit (minus the oathbearer) take that weapon.  This should not happen as the rules state "any model may exchange its bolter for the following" not "All models...."  Please change it reflect each model being able to independently take a weapon.  This doesnt affect anything else in that units profile, just the combi stuff.  Swords, shields, fists, plasma pistol etc work normally.  

**To Reproduce**
1. Go to 'Dark Angels list and select an HQ so you can then select the deathwing companions option, then add any amount for squad size.
2. Click on ' any squad member (not the oathbearer though) and then select any combi weapon option.  
4. See error as all members now have the same weapon.   A member with the shield does not have the bolter yet this makes him have it along with equiping all members of the squad with the same weapon.  

**Expected behavior**
Each member should be able to switch their bolter independently of any other model in the squad to take whichever upgrade is listed.

**Screenshots**

**Device and data (please complete the following information):**
 - OS: Android
 - BattleScribe version v2.03.25.338
 - Release version HH V16.1.4

**Comment by revivedtitan** [comment]
Good find. Will fix for the next release

**Comment by warpspyder** [comment]
Thanks. While running through it my rapiers don't have an option yet to choose their wing either for the DA.  Was building a 3k list so just noticed these things today.  On Sep. 4, 2022 14:01, revivedtitan ***@***.***> wrote:
Good find. Will fix for the next release

**Comment by revivedtitan** [comment]
Yeah hex wing is definitely being added in the next release. I went though earlier this week and added it to all the units it was missing from.

**Comment by warpspyder** [comment]
I saw alot of updates and it's great.  Solid work for sure.  Can't wait for the RoWs to show up so I can make my lists legal lol. No clue which hex wing to make grav rapiers but whatever.  Thanks again. On Sep. 4, 2022 14:10, revivedtitan ***@***.***> wrote:
Yeah hex wing is definitely being added in the next release. I went though earlier this week and added it to all the units it was missing from.
